docs/rancher/virtualization-management.md

```diff
@@ -121,6 +121,26 @@ Please refer to the [Harvester & Rancher Support Matrix](https://www.suse.com/su
 </TabItem>
 </Tabs>
 
+### Upgrades
+
+To upgrade an imported Harvester cluster, you must upgrade Rancher, the Harvester UI Extension, and Harvester in a specific order. The [Harvester UI Extension](./harvester-ui-extension.md) is required to access the Harvester UI in Rancher v2.10.x and later versions.
+
+1. Check the [support matrix](./harvester-ui-extension.md#support-matrix) to determine the Rancher and Harvester UI Extension versions that match the Harvester cluster.
+
+1. Upgrade [Rancher](https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades).
```

Comment:
If users choose to upgrade Rancher later, it is still allowed. However, users will not be able to utilize the latest UI extension with the upgrading Harvester version. New UI features are only available until users upgrade the UI extension.

We might need to clarify this a bit. cc @Yu-Jack @harvester/release

Reply:
Actually, we had written this part already. If you scroll down a litter bit, you'll see it.